phpstan.neon

```diff
@@ -303,3 +303,84 @@ parameters:
                 - packages/monorepo-builder/packages/merge/src/ComposerKeyMerger/AbstractComposerKeyMerger.php # 15
                 - packages/monorepo-builder/packages/merge/src/ComposerKeyMerger/AbstractComposerKeyMerger.php # 20
                 - packages/monorepo-builder/packages/merge/tests/ComposerJsonDecorator/AbstractComposerJsonDecoratorTest.php # 18
+
+        -
+            message: '#Method or Function call in foreach is not allowed#'
```

Comment:
Could you include fix for this here as well, so we don't have such a long list in here?

Reply:
I will try.

Reply:
implemented.